src/tcgwars/logic/impl/gen3/Deoxys.groovy

```diff
@@ -1259,10 +1270,7 @@ public enum Deoxys implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               damage 10
-              if(opp.hand){
-                opp.hand.select(hidden:true,"Select one card").showToMe("Selected card.").moveTo(opp.deck)
-                shuffleDeck(null,TargetPlayer.OPPONENT)
-              }
+              afterDamage { astonish() }
```

Comment:
Just leaving this as a note of sorts. `astonish()` explicitly mentions **Astonish**

Reply:
More fixes for statics! Hadn't noticed this one though, thanks for pointing it out (there's multiple non-astonish attacks using it already across sets)